docs/changes/units/16729.feature.rst

```diff
@@ -0,0 +1 @@
+Added SI-units ``sievert``, ``gray``, ``katal`` and ``hectare`` in ``astropy.units.si``.
```

Comment:
Nitpick: Oxford comma!
```suggestion
Added SI-units ``sievert``, ``gray``, ``katal``, and ``hectare`` in ``astropy.units.si``.
```

Reply:
Took the liberty to just push a commit for this. Hope you don't mind!

Reply:
Sure, I was about to push that and I was wondering why it failed.
But that you did it first explains it.

Thank you for your help.